packages/manager/src/features/CloudPulse/Dashboard/CloudPulseDashboard.tsx

```diff
@@ -51,6 +51,11 @@ export interface DashboardProperties {
    * optional flag to check whether changes should be stored in preferences or not (in case this component is reused)
    */
   savePref?: boolean;
+
+  /**
+   * Selected tags for the dashboard
+   */
+  tags?: string[];
```

Comment:
why we need to make this go as a separate filter here, we have additionalFilters above, it can go as a part of that right?

Reply:
we don't need it as a separate filter here, right. should we then remove region also?

Reply:
sorry, I need to take this back, this approach is correct because, additional filters is for metrics call only